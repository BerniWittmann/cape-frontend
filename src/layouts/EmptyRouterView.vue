<template>
  <router-view></router-view>
</template>

<script>
/* ============
 * Empty Router View
 * ============
 *
 * Renders an empty Router View
 */

export default {}

</script>
