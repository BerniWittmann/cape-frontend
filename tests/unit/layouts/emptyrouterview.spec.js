import { shallowMount } from '@vue/test-utils'
import { i18n } from '../setupPlugins'

import EmptyRouterView from '@/layouts/EmptyRouterView.vue'
import EmptySlotComponent from '../EmptySlotComponent.vue'

describe('Layouts', () => {
  describe('EmptyRouterView', () => {
    it('renders', () => {
      const wrapper = shallowMount(EmptyRouterView, {
        i18n,
        stubs: {
          'router-view': EmptySlotComponent
        }
      })
      expect(wrapper.html()).toMatchSnapshot()
    })
  })
})
