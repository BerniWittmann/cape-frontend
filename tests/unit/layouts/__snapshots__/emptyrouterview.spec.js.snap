// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Layouts EmptyRouterView renders 1`] = `
<div class="empty-router-view">
  <div name="dialog"> </div>
  <div> </div>
</div>
`;
